### PR TITLE
Fix: broken routing when logged out

### PR DESCRIPTION
**Pull request by kms-rscholz**
@johannschopplich 

I’ve just realised that the proposed draft preview fix earlier introduced a bug for logged out users, where all pages redirected to the error page.

Here `isReadable()` is replaced by `isVerified()`. In order to make this work, the query token needs to be attached to all `.json` requests as well. Otherwise the data could either not be fetched or would be publicly exposed.

https://getkirby.com/docs/reference/objects/cms/page/is-verified

Sorry for introducing the bug prior. This should fix it.

### CODE REVIEW

File: src/hooks/useKirbyApi.js [hunk 1]
```diff
@@ -50,10 +50,12 @@ const apiUri = (path) => {
  * @param {boolean} [options.revalidate=false] Skip cache look-up and fetch page freshly
```
Comment (johannschopplich) [inline src/hooks/useKirbyApi.js:50]: Could you add the optional `token` parameter description, please?

```
* @param {string} [options.token] Description …
```

File: src/hooks/useKirbyApi.js [hunk 2]
```diff
@@ -50,10 +50,12 @@ const apiUri = (path) => {
...
  * @returns {Promise<object|boolean>} The page's data or `false` if fetch request failed
  */
-const getPage = async (id, { revalidate = false } = {}) => {
+const getPage = async (id, { revalidate = false, token = null } = {}) => {
```
Comment (johannschopplich) [inline src/hooks/useKirbyApi.js:53]: I prefer `undefined` instead of `null`. Thus, `{ revalidate = false, token }` should be sufficient.

**Review by johannschopplich — COMMENTED** [review]
I see, good catch from your previous PR. Looks almost good to me; see comments.

**Comment by kms-rscholz** [comment]
@johannschopplich done, sorry again for the broken pull request prior!

**Comment by johannschopplich** [comment]
It's fine — should've checked myself. So it's on us both. 😉 Thanks for the overhaul!

**Pull request merged**